NitroxModel/Helper/NitroxUser.cs

```diff
@@ -15,6 +16,15 @@ public static class NitroxUser
         private const string PREFERRED_GAMEPATH_REGKEY = @"SOFTWARE\Nitrox\PreferredGamePath";
         private static string launcherPath;
         private static string gamePath;
+        private static LinuxConfig _linuxConfig;
+        public static LinuxConfig linuxConfig {
+            get {
+                if (_linuxConfig == null) {
+                    _linuxConfig = LinuxConfig.Load();
+                }
+                return _linuxConfig;
+            }
+        }
```

Comment:
Does not conform code style for braces on separate lines and variable naming (no `_`). Or could be rewritten as:
```suggestion
        private static LinuxConfig linuxConfig;
        public static LinuxConfig LinuxConfig => linuxConfig ??= LinuxConfig.Load();
```

Reply:
Addressed in newest commits.